Introduce FAQ section into kernelCTF guidelines. (#133)
google/security-research: kernelctf/faq.md

# Frequently Asked Questions (FAQ)

> #### 1. I'd sent one [bughunters.google.com](https://bughunters.google.com) report which contains multiple submissions (exploits). I'd noticed that only some PRs associated with the report are merged. After that I received a bounty email from kernelCTF which awards partial bounty (bounty not for all exploits submitted). When would I receive all the bounties?

Reviewer: ```suggestion
#### 1. I submitted one report to [bughunters.google.com](https://bughunters.google.com/) containing multiple exploits. I noticed that some of the associated pull requests have been merged, and I received a payment notification email from kernelCTF awarding only partial rewards (not for all submitted exploits). When will I receive all the bounties?
```

> The above described situation means that at this moment only some exploits were reviewed, the rest of exploits associated with the report still stay in the review queue. We're doing everything possible to prioritize review of similar submissions and bundle them together, but sometimes we still need more time to make a decision on all exploits provided. Please stay tuned for the updates on the rest of pull requests. As soon as pull requests are merged the subsequent bounties will be issued. 

Reviewer: ```suggestion
The above described situation means that at this moment only some of the exploits have been reviewed, the rest of exploits associated with the report remain in the review queue. We are doing everything possible to prioritize review of similar submissions and bundle them together, but sometimes we still need more time to make a decision on all the exploits provided. Please stay tuned for the updates on the remaining pull requests. As soon as other pull requests are merged the subsequent bounties will also be issued. 
```
